submodules/tags/main.tf

```diff
@@ -12,6 +12,9 @@ locals {
     "aws/account_number" = data.aws_caller_identity.current.account_id
     "octopus/project"    = var.octopus_tags.project
     "octopus/space"      = var.octopus_tags.space
+    "octopus/environment_name"      = var.octopus_tags.environment_name
```

Comment:
Why do you add _name everywhere? It is not added in the existing ones.

Reply:
Story says Name in the key. I can remove, if it is not needed.

Octopus.Environment.Name
Octopus.ProjectGroup.Name
Octopus.Release.Channel.Name

Reply:
@naveen-variant Does `octopus/project` or `octopus/space` have a name in their key? If not, then why is there `_name` in others? What is the Variable in octopus that has the value of `octopus/project` or `octopus/space` stored? Please find that and comment here.

Reply:
Looks like .Name is reference from octopus. Removing _name from the keys.
octopus variable for project and space.
  space_name=$(get_octopusvariable "Octopus.Space.Name")
  project_name=$(get_octopusvariable "Octopus.Project.Name")

  export TF_VAR_octopus_tags='{ project = "'$project_name'", space = "'$space_name'" }'